### PR TITLE
feat(chat): improve UX when user tries to copy paste images

### PR DESCRIPTION
**Pull request by nickjjang**
AP-93

### CODE REVIEW

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
This works very well. Can you make one change though:

In Upload.vue, can you add the `declare module 'vue/types/vue' {` snippet back in, and then add the things to it/remove the ts-ignores?

I think it will look something like 
```
declare module 'vue/types/vue' {
  interface Vue {
    loadPicture: (item: UploadDropItemType) => void,
    cancelUpload: () => void,
    finishUploads: () => void,
    dispatchFile: (file: FileType) => void,
  }
}
```

**Comment by nickjjang** [comment]
> **WanderingHogan ** requested changes

I have  fixed it.

File: components/views/chat/chatbar/Chatbar.vue [hunk 1]
```diff
@@ -150,10 +151,10 @@ export default Vue.extend({
...
      */
     handleChatBorderRadius() {
-      const wrap = this.$refs.wrap as HTMLElement
-      if (wrap.offsetHeight > 50 || this.ui.replyChatbarContent.id)
-        wrap.style.borderRadius = '4px'
-      else wrap.style.borderRadius = '41px'
+      // const wrap = this.$refs.wrap as HTMLElement
+      // if (wrap.offsetHeight > 50 || this.ui.replyChatbarContent.id)
+      //   wrap.style.borderRadius = '4px'
+      // else wrap.style.borderRadius = '41px'
```
Comment (stavares843) [inline components/views/chat/chatbar/Chatbar.vue:157]: commented-out code 🔨

File: components/views/files/upload/Upload.html [hunk 2]
```diff
@@ -12,69 +12,90 @@
...
+          :action="sendMessage"
+        />
         <InteractablesButton
           type="danger"
           size="small"
           :text="$t('pages.files.upload.cancel')"
           :action="cancelUpload"
         />
-      </div>
+      </div> -->
```
Comment (stavares843) [inline components/views/files/upload/Upload.html:90]: commented-out code 🔨

**Review by stavares843 — APPROVED** [review]
pretty neat! 🎉

**Review by WanderingHogan — APPROVED** [review]

**Pull request merged**